docs/source/datasheet/simple_registers/clk_divider.rst

```diff
@@ -0,0 +1,56 @@
+.. _datasheet_simple_registers_blinking:
+
+Clock Divider
+--------
+
+Introduction
+~~~~~~~~~~~~
+
+This benchmark is designed to test the flip-flop/registers in FPGAs.
+The benchmark takes an input clock signal clk_i and generates an output clock signal clk_o with one fourth the frequency of the input clock. The module also has a reset input rst to reset output signals.
+
+Source codes
+~~~~~~~~~~~~
+
+See details in ``simple_registers/clk_divider``
+
+Block Diagram
+~~~~~~~~~~~~~
+
+.. figure:: ./figures/clk_divider.svg
+  :width: 60%
+  :alt: Clock divider schematic
+
+  Clock divider schematic
+
+
+Performance
+~~~~~~~~~~~
+
+Expect to consume only 1 LUT and 1 flip-flop of an FPGA.
```

Comment:
I think it should require at least 2 LUTs and 2 FFs, can you double check?

Reply:
@tangxifan  Yes, expectation is 2. I will see and update what it actually takes while adding it in alkaid_efpga dv flow.